packages/react/src/date-picker/Calendar.stories.tsx

```diff
@@ -2,20 +2,25 @@ import { Meta, StoryObj } from '@storybook/react';
 import { useState } from 'react';
 import { DateRange } from 'react-day-picker';
 import { subDays, addDays, getYear } from 'date-fns';
-import { CalendarRange, CalendarSingle } from './Calendar';
+import { CalendarRange, CalendarSingle, CalendarSingleProps } from './Calendar';
+import { CalendarProvider, CalendarProviderProps } from './CalendarContext';
 
-const meta: Meta<typeof CalendarSingle> = {
+const meta: Meta<CalendarSingleProps & CalendarProviderProps> = {
 	title: 'forms/DatePicker/Calendar',
 	component: CalendarSingle,
-	render: function Render(props) {
+	render: function Render({ yearRange, ...props }) {
 		const [value, setValue] = useState<Date>();
-		return <CalendarSingle selected={value} onSelect={setValue} {...props} />;
+		return (
+			<CalendarProvider yearRange={yearRange}>
```

Comment:
We need to wrap the calendar in storybook and tests with the new context provider now that's not part of this component anymore.